addon/components/o-s-s/star-rating.ts

```diff
@@ -38,18 +45,37 @@ export default class OSSStarRating extends Component<OSSStarRatingArgs> {
   }
 
   get activeColorClass(): string {
-    return `color-${this.args.activeColor || 'yellow'}`;
+    return `${this.args.onChange && 'oss-star-rating'} color-${this.args.activeColor || 'yellow'}`;
   }
 
   get passiveColorClass(): string {
-    return `color-${this.args.passiveColor || 'grey'}`;
+    return `${this.args.onChange && 'oss-star-rating'} color-${this.args.passiveColor || 'grey'}`;
```

Comment:
oh, I saw the less code, so this is to prevent having a pointer cursor when using the Attribute::Rating, right ?
I think it would be better to restore the orginal getters here and to add a `splattribute` to the `o-s-s/star-rating.hbs` at the top level.
This way, from the `OSS::Attribute::Rating` you can pass a new class, something like "no-hover-pointer" and deactivate it from the parent.